Jenkinsfile

```
@@ -0,0 +1,78 @@
+@Library('jenkins.shared.library') _
+
+pipeline {
+ agent {
+   label 'ubuntu_docker_label'
+ }
+ environment {
+   HELM_IMAGE = "infoblox/helm:3.2.4-5b243a2"
```

This version is EOL. Switch to a newer version. `helm:3` will track the latest.
```suggestion
   HELM_IMAGE = "infoblox/helm:3"
```

---

I have updated helm version, please check